src/ui/components/ObjectView.vue

```diff
@@ -272,9 +272,7 @@ export default {
         this.loadComposition();
       }
 
-      this.viewContainer = document.createElement('div');
-      this.viewContainer.classList.add('l-angular-ov-wrapper');
```

Comment:
Does this class not have any associated styles? I know at some point we had some old classes with styles that were still needed.

Reply:
I removed this with @charlesh88. It’s old and no longer needed